zbus/src/connection/mod.rs

```diff
@@ -1722,6 +1722,52 @@ mod p2p_tests {
         )
     }
 
+    #[cfg(any(
+        all(feature = "vsock", not(feature = "tokio")),
+        feature = "tokio-vsock"
+    ))]
+    #[test]
+    #[timeout(15000)]
+    fn vsock_connect() {
+        crate::utils::block_on(async {
```

Comment:
Last nitpick, I promise. :) If you look at other tests, I've been splitting the async part into a separate function and then block_on on the whole function call. That avoid unnecessary indentation level, that always makes the code a bit harder to read.